taxi/models.py

```diff
@@ -0,0 +1,51 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63)
+    country = models.CharField(max_length=63)
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["name", ],
+                                    name="unique_manufacturer_name"
+                                    ),
+        ]
+        ordering = ("name", )
+
+    def __str__(self):
+        return "{} ({})".format(self.name, self.country)

```

Comment:
This is a minor issue, but it is recommended not to overuse f-strings or format methods when they are not necessary. Consider using a simple string concatenation or f-string directly.